test/test_smbo/test_epm_configuration_chooser.py

```diff
@@ -0,0 +1,182 @@
+import sys
+import unittest
+from unittest import mock
+
+import numpy as np
+from ConfigSpace import Configuration
+
+from smac.epm.rf_with_instances import RandomForestWithInstances
+from smac.facade.smac_ac_facade import SMAC4AC
+from smac.optimizer.objective import average_cost
+from smac.runhistory.runhistory import RunHistory
+from smac.scenario.scenario import Scenario
+from smac.utils import test_helpers
+from smac.tae.execute_ta_run import StatusType
+
+
+class ConfigurationMock(object):
+    def __init__(self, value=None):
+        self.value = value
+
+    def get_array(self):
+        return [self.value]
+
+
+class TestEPMChooser(unittest.TestCase):
+
+    def setUp(self):
+        self.scenario = Scenario({'cs': test_helpers.get_branin_config_space(),
+                                  'run_obj': 'quality',
+                                  'output_dir': 'data-test_epmchooser'})
+
+    def branin(self, x):
+        y = (x[:, 1] - (5.1 / (4 * np.pi ** 2)) * x[:, 0] ** 2 + 5 * x[:, 0] / np.pi - 6) ** 2
+        y += 10 * (1 - 1 / (8 * np.pi)) * np.cos(x[:, 0]) + 10
+
+        return y[:, np.newaxis]
+
+    def test_choose_next(self):
+        seed = 42
+        config = self.scenario.cs.sample_configuration()
+        rh = RunHistory(aggregate_func=average_cost)
+        rh.add(config, 10, 10, StatusType.SUCCESS)
+
+        smbo = SMAC4AC(self.scenario, rng=seed, runhistory=rh).solver
+
+        x = next(smbo.epm_chooser.choose_next()).get_array()
+        self.assertEqual(x.shape, (2,))
+
+    def test_choose_next_w_empty_rh(self):
+        seed = 42
+        smbo = SMAC4AC(self.scenario, rng=seed).solver
+        smbo.runhistory = RunHistory(aggregate_func=average_cost)
+
+        # should return random search configuration
```

Comment:
Could you please check the value of `x.origin`? It should be something like 'random search'.